TINY-7476: Added table cell class and table class. (#6841)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

### 5.8.1 - TBD

> ### Added

Reviewer: This should be in the `Unreleased` section which is what will be used for 5.9.

- [ ] Move to correct section

> - Added the new command `mceTableCellToggleClass` which toggles currently selected cells to have or not have the provided class #TINY-7476

Reviewer: ```suggestion
- Added a new `mceTableCellToggleClass` command which toggles the provided class on the currently selected cells #TINY-7476
```

> - Added the new command `mceTableCellToggleClass` which toggles currently selected cells to have or not have the provided class #TINY-7476

Reviewer 2: Oops, old comments. Ignore.

> - Added the new command `mceTableToggleClass` which toggles currently selected table to have or not have the provided class #TINY-7476

Reviewer: ```suggestion
- Added a new `mceTableToggleClass` command which toggles the provided class on the currently selected table #TINY-7476
```

> - Added the new command `mceTableToggleClass` which toggles currently selected table to have or not have the provided class #TINY-7476

Reviewer 2: Oops, old comments. Ignore.
